ui/app/components/global-search/control.js

```diff
@@ -180,14 +181,20 @@ export default class GlobalSearchControl extends Component {
   @action
   selectOption(model) {
     if (model.type === 'job') {
-      this.router.transitionTo('jobs.job', model.id, {
-        queryParams: { namespace: model.namespace },
+      const fullId = JSON.stringify([model.id, model.namespace]);
+      this.store.findRecord('job', fullId).then((job) => {
+        this.router.transitionTo('jobs.job', job.idWithNamespace);
       });
```

Comment:
If we extract the logic to generate the proper namespaced job ID into a helper function we could avoid this store query, but I think it's fine this way?